Documentation/Extbase/Step3Documentation/Bootstrap.rst

```diff
@@ -32,8 +32,8 @@ global TypoScript template::
    pluginName = Products
  }
 
-As soon as you add this Extbase plugin to a page, the method *run* in the class *tx_extbase_core_bootstrap* is 
-exectued. So that it knows wich extension and plugin we're dealing with, information about the *extensionName* and 
+As soon as you add this Extbase plugin to a page, the method *run* of the class *tx_extbase_core_bootstrap* is being 
```

Comment:
I'd say this:
... *run* by the class *tx_extbase_core_bootstrap* ...

Reply:
this is merged would you make a new PR about that @DavidBruchmann?